packages/2d/src/components/Txt.ts

```diff
@@ -105,6 +105,20 @@ export class Txt extends Shape {
     context.restore();
   }
 
+  protected override getCacheBBox(): BBox {
+    const cache = super.getCacheBBox();
+
+    if (!this.element.hasChildNodes) return cache;
+
+    const size = this.computedSize();
+    const range = document.createRange();
+    range.setStartBefore(this.element.firstChild!);
+    range.setEndAfter(this.element.lastChild!);
```

Comment:
Wouldn't this work?
```ts
range.selectNodeContents(this.element);
```

Reply:
I'll try it again, but I recall when I tested that the width of the div changed the result.